client/lib/feed-stream-store/actions.js

```diff
@@ -11,6 +11,8 @@ import { action as ActionType } from './constants';
 import FeedPostStoreActions from 'lib/feed-post-store/actions';
 import feedPostListCache from './feed-stream-cache';
 import wpcom from 'lib/wp';
+import { reduxDispatch } from 'lib/redux-bridge';
```

Comment:
I think i'd prefer to just call it `dispatch`, but i guess that doesn't matter.
any file that uses this bridge will never use the real one, we'll probably need to write wholly new files anway.